test-lib-openshift.sh

```diff
@@ -273,6 +273,13 @@ function ct_os_new_project() {
   oc new-project ${project_name}
   # let openshift cluster to sync to avoid some race condition errors
   sleep 3
+  if test -n "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}" -a -e "${OPENSHIFT_CLUSTER_PULLSECRET_PATH:-}"; then
```

Comment:
I think before this line, there could be a short explanation of what this variable means (as it is not described elsewhere). E.g. `OPENSHIFT_CLUSTER_PULLSECRET_PATH variable contains a path to the k8s secret definition that is later used for pulling images from registries that need authentication to access (eg. registry.redhat.io)`

Reply:
Good point, will add it to the docstring of the function.